seeds/_sector/dex/wombat_exchange_optimism_base_trades_seed.csv

```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+optimism,wombat_exchange,2,2024-02-15,0x6a5db60607b8aad885506fa8766a135cc3d715e27f8d4df037bf232901b21fb6,117,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0x7f5c764cbc14f9669b88837ca1490cca17c31607,116190194,1357692117,1358827712
+optimism,wombat_exchange,2,2024-02-15,0xd105dc0022b73ac1ff31568a79f3a978dd72c957cd09e8bdd7f3a5d558f87543,16,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0x7f5c764cbc14f9669b88837ca1490cca17c31607,116190025,1477000000,1478437059
```

Comment:
https://optimistic.etherscan.io/tx/0xd105dc0022b73ac1ff31568a79f3a978dd72c957cd09e8bdd7f3a5d558f87543

From the explorer:
token sold: USDT
token bought: USDC.e